ramalama/shortnames.py

```diff
@@ -0,0 +1,32 @@
+import configparser
+import os
+
+
+class Shortnames:
+    """Shortnames utility class"""
+
+    shortnames = {}
+
+    def __init__(self):
+        file_paths = [
+            "/usr/share/ramalama/shortnames.conf",
+            "/etc/ramalama/shortnames.conf",
+            os.path.expanduser("~/.config/ramalama/shortnames.conf"),
+            "./shortnames/shortnames.conf",  # for development
+            "./shortnames.conf",  # for development
+        ]
+
+        for file_path in file_paths:
```

Comment:
If we do this:

```
        for file_path in file_paths:
            config = configparser.ConfigParser(delimiters=('='))  # Only use '=' as a delimiter
            config.read(file_path)
            if "shortnames" in config:
                self.shortnames.update(config["shortnames"])
```

I think we can have shortnames in this format:  "granite-lab:7b", "merlinite-lab:7b"

But I haven't tested. It wasn't working before because python's config parser by default treats both '=' and ':'  as delimiters by default.

But we can do it in the next PR.

The colon separator for the 3b, 7b, etc. was popularised by Ollama.